src/pheval/analyse/generate_plots.py

```diff
@@ -95,55 +121,46 @@ def generate_stacked_bar_plot(
     def _generate_cumulative_bar_plot_data(self, benchmark_result: BenchmarkRunResults):
         """Generate data in correct format for dataframe creation for cumulative bar plot."""
         rank_stats = benchmark_result.rank_stats
-        trimmed_corpus_results_dir = trim_corpus_results_directory_suffix(
-            benchmark_result.results_dir.name
-        )
+        run_identifier = self.create_run_identifier(benchmark_result.results_dir)
```

Comment:
Can this only ever be a directory path? I don't think they should be conflated. For example what if this all ran off an embedded SQLite database where the run identifier would be better off as a row id or if this should be changed for publication display or simply for having a more compact and easily identifiable identifier?